### ISSUE TITLE
[StarBook mkVI] Warning in the logs "ACPI: battery: [Firmware Bug]: (dis)charge rate invalid"

### ISSUE THREAD

**Issue by sim31** [StarLabsLtd/firmware#216]
I see this in logs on every boot:

```
kernel: ACPI: battery: [Firmware Bug]: (dis)charge rate invalid.
```

Wondering what this means and is it safe to ignore?

**Comment by Sean-StarLabs** [comment]
What kernel, fw versions, charge level, and is the charger connected?

**Comment by sim31** [comment]
Kernel: 6.10.10
Coreboot 24.03
EC: 24.3
Charge level is around 80%. The max charge level is set to 80%.

Charger is typically connected (USB-c) but today I tried booting without charger connected and I see the same warning.

**Comment by Sean-StarLabs** [comment]
Ah yeah, fixed that a little while ago - fix in the next release, but it's not major

**Comment by sim31** [comment]
Thanks. Looking forward to the next release.

**Comment by Sean-StarLabs** [comment]
Fixed with adf1039e83eeb8f32c773488cf21d474c53ff8af
